packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -284,6 +284,38 @@ void main() {
     skip: kIsWeb, // [intended] the web handles this on its own.
   );
 
+  testWidgets('can get text selection color initially on desktop', (WidgetTester tester) async {
+    final FocusNode focusNode = FocusNode();
+    final TextEditingController controller = TextEditingController(
+      text: 'blah1 blah2',
+    );
+    await tester.pumpWidget(
+      CupertinoApp(
+        home: Center(
+          child: RepaintBoundary(
+            child: CupertinoTextField(
+              key: const ValueKey<int>(1),
+              controller: controller,
+              focusNode: focusNode,
+            ),
+          ),
+        ),
+      ),
+    );
+
+    controller.selection = const TextSelection(baseOffset: 0, extentOffset: 11);
+    focusNode.requestFocus();
+    await tester.pump();
+
+    expect(focusNode.hasFocus, true);
+    expect(
+      find.byKey(const ValueKey<int>(1)),
+      matchesGoldenFile('text_field_golden.text_selection_color.0.png'),
+    );
+  },
+    variant: TargetPlatformVariant.desktop(),
```

Comment:
nit: the indentation seems off. Also you probably don't need the variant anyways.